### DISCUSSION TITLE
The Wave and MIDI driver requires a newer version of the virtual DMA driver.

### DISCUSSION THREAD

**Discussion by jschwartzenberg** [dosemu2/dosemu2#2252] [General]
Does anybody know what this means? Where could I get this driver?
![Screenshot_20240806_212039](https://github.com/user-attachments/assets/db04b891-3a16-4a70-8462-0ee933cd627f)

Note that I do not get this message when starting Windows in standard mode. Audio works then.

**Comment by stsp** [comment]
You shouldn't use creative SB16 drivers.
Use windows-provided older SB drivers.

**Reply by jschwartzenberg** [reply]
Yeah I know those always work, but they don't support higher sampling rates. There are much better drivers out there especially those meant for the SB16 and most of them do work in both standard and enhanced mode. I just thought maybe somebody could help me with fixing this specific driver set as I think it's a newer one.

**Reply by stsp** [reply]
Does it have some new .386 file
that wasn't there before? If so,
there is probably "no fix".

**Reply by jschwartzenberg** [reply]
Yes, I guess so. That's good to know thanks! I'll just look at using another driver.
